src/test/java/com/epam/tc/steps/AssertionSteps.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.tc.steps;
+
+import com.epam.tc.data.DataToRemove;
+import com.epam.tc.dto.BoardDto;
+import com.epam.tc.dto.LabelDto;
+import com.epam.tc.dto.ListDto;
+import org.junit.Assert;
+
+public class AssertionSteps {
+
+    public static void verifyThatBoardExist(String expectedBoardName) {
```

Comment:
This check always passes, even if I try to check that board with some random name exists